.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+
+Tell us what this change does. If you're fixing a bug, please mention
+the github issue number.
```

Comment:
Not a bad idea, though most PRs I've seen for metasploit-payloads have already done this, so not a huge problem.